v5/tutorials/topical/display.rst

```diff
@@ -0,0 +1,16 @@
+=======================
+V5 Brain Display (LVGL)
+=======================
+
+Interacting with the touchscreen on the V5 Brain is made possible through `LVGL <https://littlevgl.com>`_.
+LVGL is a full-featured C graphics library (it's accessible in C++ projects too under the same API).
+
+The first step to getting started with LVGL is to include ``apix.h`` in your ``main.h`` file or other header files.
+This includes the full LVGL feature set as described in their documentation: https://littlevgl.com/basics

```

Comment:
Does this link end up as a hyperlink?

Reply:
yep! Anything starting with `http(s)://` gets turned into a link by Sphinx